app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">AndroidTrainning</string>
+    <string name="first_number">so 1</string>
+    <string name="second_number">so 2</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.second_number` appears to be unused
 Unused resources make applications larger and slow down builds.